### PR TITLE
Docblock links

### PR DESCRIPTION
**Pull request by Camwyn**
Replace placeholder in v2 and blocks templates with link for v2: https://theeventscalendar.com/knowledgebase/k/calendar-template-files-v2/?utm_medium=plugin-tec&utm_campaign=in-app 

Replace placeholders in legacy templates with a link for v1: https://theeventscalendar.com/knowledgebase/stylesheets-and-page-templates/?utm_medium=plugin-tec&utm_campaign=in-app

### CODE REVIEW

File: src/views/blocks/featured-image.php [hunk 1]
```diff
@@ -7,12 +7,12 @@
...
+ * @link http://m.tri.be/1aiy
  *
  * @version 4.7
  *
  */
 
 $event_id = $this->get( 'post_id' );
 ?>
-<?php echo tribe_event_featured_image( $event_id, 'full', false );
\ No newline at end of file
```
Comment (tr1b0t) [inline src/views/blocks/featured-image.php:18]: * :no_entry_sign: Opening PHP tag must be on a line by itself
* :no_entry_sign: All output should be run through an escaping function (see the Security sections in the WordPress Developer Handbooks), found 'tribe_event_featured_image'.
* :warning: Expected next thing to be an escaping function (see Codex for 'Data Validation'), not ')'

<sub>hash: 7f65f3208cf20a86c2ecdcb9a04438bd</sub>

File: src/views/blocks/featured-image.php [hunk 2]
```diff
@@ -7,12 +7,12 @@
...
  *
  * @version 4.7
  *
  */
 
 $event_id = $this->get( 'post_id' );
-?>
-<?php echo tribe_event_featured_image( $event_id, 'full', false );
\ No newline at end of file
+
```
Comment (tr1b0t) [inline src/views/blocks/featured-image.php:17]: * :no_entry_sign: All output should be run through an escaping function (see the Security sections in the WordPress Developer Handbooks), found 'tribe_event_featured_image'.
* :warning: Expected next thing to be an escaping function (see Codex for 'Data Validation'), not ')'

<sub>hash: 68b46e3aafc8ba116032007f799e8baa</sub>

File: src/views/single-event-blocks.php [hunk 3]
```diff
@@ -9,7 +9,7 @@
  *
  * See more documentation about our Blocks Editor templating system.
  *
- * @link {INSERT_ARTCILE_LINK_HERE}
+ * @link http://m.tri.be/1ami+
```
Comment (paulmskim) [inline src/views/single-event-blocks.php:12]: you have a handful of these URLs that aren't correct.

File: tests/views_integration/Tribe/Events/Views/V2/Template/__snapshots__/TitleTest__test_day_view__1.php [hunk 4]
```diff
@@ -1 +1 @@
-<?php return 'Upcoming Events';
+<?php return 'Events for August 12, 2020 - August 12, 2020';
```
Comment (lucatume) [inline tests/views_integration/Tribe/Events/Views/V2/Template/__snapshots__/TitleTest__test_day_view__1.php:1]: This is the date of the snapshot generation, this means some date-related value is not mocked correctly.
This must be fixed.

Comment (Camwyn) [reply tests/views_integration/Tribe/Events/Views/V2/Template/__snapshots__/TitleTest__test_day_view__1.php:1]: This text looks wrong as well?

**Comment by lucatume** [comment]
REST API v1 tests are failing due to timeouts, not changes to the code. What tests had issues have been fixed. Further look into REST API v1 tests will be part of another PR.

**Pull request merged**